db/migrate/20170720125756_create_product_nature_variant_valuings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProductNatureVariantValuings < ActiveRecord::Migration
+  def change
+    create_table :product_nature_variant_valuings do |t|
```

Comment:
Table is here for historic, so we need to add an explicit column to track event times like: `computed_at`